app/pages/SchoolPage.js

```diff
@@ -60,7 +62,7 @@ class SchoolPage extends React.Component {
           <SchoolTitle name={this.state.mainName} />
```

Comment:
This happens because the first time the page is rendered there is no school, so mainName is an empty object and official_name is undefined, Should I solve this creating default data, or avoiding the calling to SchoolTitle if there is not school?